packages/sit-onyx/src/components/OnyxListbox/OnyxListbox.stories.ts

```diff
@@ -22,8 +23,42 @@ const meta: Meta<typeof OnyxListbox> = {
   title: "support/OnyxListbox",
   ...defineStorybookActionsAndVModels({
     component: OnyxListbox,
-    events: ["update:modelValue"],
+    events: ["update:modelValue", "loadMore"],
+    argTypes: {
+      loadingMode: {
+        options: LOADING_MODES,
+        control: { type: "radio" },
+      },
+    },
   }),
+  /**
+   * Decorator that simulates the load more functionality so we can show it in the stories.
+   */
+  decorators: [
```

Comment:
with the decorator, the whole loading more options thing is not visible in the code preview.
I guess it's ok for the demo purpose, it would just be even nicer if the code preview could show something that can actually just be copied with a <script> section ^^ #future music

<img width="1069" alt="image" src="https://github.com/SchwarzIT/onyx/assets/151019977/81be757a-fa56-40a8-81f1-de497e1e315d">

Reply:
I updated the code snippets, can you please check again?
I am not sure if we can show a `<script>` section with the actual function that generates more options